docs/src/conf.py

```diff
@@ -358,6 +358,7 @@ def _dotv(version):
     "http://www.esrl.noaa.gov/psd/data/gridded/conventions/cdc_netcdf_standard.shtml",
     "http://www.nationalarchives.gov.uk/doc/open-government-licence",
     "https://www.metoffice.gov.uk/",
+    "http://www.metarelate.net/",
```

Comment:
Some of us have discussed removing this link entirely (it's just in a What's New), since the website is unlikely to come back.

Reply:
We don't _have_ to hold up PRs with this sort of thing 😉 #4140